Final Project App/Final Project App/ViewController.swift

```diff
@@ -30,24 +28,20 @@ class ViewController: UIViewController, UITableViewDataSource {
         NotificationCenter.default.addObserver(self, selector: #selector(ViewController.keyboardWillShow), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(ViewController.keyboardWillHide), name: NSNotification.Name.UIKeyboardWillHide, object: nil)
         
-        self.InputTextField.delegate = self //Assign Text Field Delegate
-        self.tableView.dataSource = self // Assign Table view data source
+        inputTextField.delegate = self //Assign Text Field Delegate
+        tableView.dataSource = self // Assign Table view data source
         getData() // Call get data function on initial load
```

Comment:
`getData()` should have a more descriptive name. `getMessagesFromServer` or something.